script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
```

Comment:
```suggestion
const toDoForm = document.querySelector(".todo-form");
const toDoInput = toDoForm.querySelector("input");
const toDoList = document.querySelector(".todo-list");
const finishedList = document.querySelector(".finished-list");
```
html 태그의 id와 class 네이밍은 `kebab-case`로 작성하는 게 일반적이라고 알고 있습니다...! (다른 분들 PR에 코멘트 달렸던 내용이기도 합니다!)